homeworks/sargis_vardanyan/23-11-2021/selectionstatements/IfElseTask4.java

```diff
@@ -0,0 +1,40 @@
+package selectionstatements;
+
+import java.util.Scanner;
+
+/*
+  Get digit with the highest value, from a given 3-digit number.
+*/
+public class IfElseTask4 {
+
+public static void main(String [] args){
+
+System.out.println("Enter a number with 3 digit");
+Scanner scanner = new Scanner(System.in);
+int number = scanner.nextInt();
+
+int max;
+int a = number % 10;
+number = number / 10;
+int b = number % 10;
+number = number /10;
+int c = number;
+ 
+if(c >= b) {
+ 
+   if(c >= a){
+      max = c;
+   } else {
+      max = a;     
+   }
+ } else if(b >= a){
+     max = b;
+ } else {
+     max = a;
+ }
```

Comment:
could you simplify it by assigning the first digit to max?

Reply:
Maybe this way?        

int max = a;

if(max >= b) {
if(c >= max){
max = c;
  }
} else if(b >= c){
 max = b;
 } else {
 max = c;
 }

Reply:
you don't need nested if statement
int max = a;
if(max > b) {
   max = b;
}
if (max > c) {
    max = c;
}

Reply:
Okay thanks